### PR TITLE
Refactor data upload: make 'statements' objects and consistently use them

### LINKED ISSUE #37: BUG: Uploading csv with index creates double index and breaks render
When downloading a fresh CSV from the [template](https://nlesc.sharepoint.com/:x:/r/sites/esi-far77/_layouts/15/Doc.aspx?sourcedoc=%7Bdea5b2eb-0e40-4558-ba3e-9ba9acc8b792%7D&action=edit&wdenableroaming=1&wdfr=1&wdlcid=en-US&wdorigin=ItemsView&wdhostclicktime=1715869016578&wdredirectionreason=Force_SingleStepBoot&wdinitialsession=66d38ec2-ab7f-5753-296a-3ee7cf6a6fc3&wdrldsc=2&wdrldc=1&wdrldr=ServiceCreationTimedOut), Excel adds a row-index by default. Our tool does not check whether an index already exists, so it adds another one, which breaks the mapping between the values and expected column names. Especially the statement type is then not found, which means the nodes are rendered in black and you can't read the text.

### PR DESCRIPTION
**Pull request by sjvrijn**
Fixes #37, (partially) addresses #43 

We used to load the data into `columnNames` and `rowValues`, which were inconsistently passed around and stored through the code.

This PR:
- Creates a single global object `window.INA`, that all 'global' variables are now a part of, making it instantly clear when we're using a global variable somewhere
- Converts every column name to "Title Case", for consistency in rendering and validity checking. This makes the `rowAttributes` array obsolete.
- Adds an if-statement to only add an `Id` column if it is not yet present
- Changes column name checking to only check for **missing** columns from those we expect, ignoring any extra columns
- Creates `statement` objects at file upload time that map each column name to the corresponding value **<== This gets rid of any fixed array order dependency and lets the rendering happen using the exact columns we want to use, regardless of their order in the file. This is what actually addresses issue #37**
- Changes the session storage and table/chart re-rendering to use the plain `columnNames` array and the array of `statements` objects

As a result, this removes any global references to `rowValues`, since everything is replaced with the previously mentioned `statements` objects instead.

### CODE REVIEW

File: includes/uploader.js [hunk 1]
```diff
@@ -161,18 +162,27 @@ function checkFileContent() {
...
-            // Extract and log each row
-            rowValues = content.split('\n').slice(1).map(row => row.split(','));
-
-            // Add an ID column in front
-            columnNames.unshift("ID");
-            let rowsWithID = rowValues.map((row, index) => [(index + 1)].concat(row));
-
-            rowValues = rowsWithID;
-            window.rowValues = rowsWithID;
+            let lines = content.split('\n');
```
Comment (suvayu) [inline includes/uploader.js:165]: IIUC, this is parsing the uploaded file into rows. Will this break if there are embedded newlines in cell contents? Excel users often have multiline strings when working with text in spreadsheets. Can't we use a standard library for this kind of stuff?

Comment (ermannolocascio) [reply includes/uploader.js:165]: Wwe should make a test. I don't know.

**Review by ermannolocascio — APPROVED** [review]
It works fine. The code is also fine to my eyes. Thank you.

File: includes/uploader.js (hunk 1 above)
Comment (sjvrijn) [reply includes/uploader.js:165]: I agree it's something we should look into, I've opened issue #64 on it.

**Pull request merged**